diff -u pandoc-1.9.1.2/templates/default.latex\~ pandoc-1.9.1.2/templates/default.latex
--- pandoc-1.9.1.2/templates/default.latex~	2012-03-10 07:53:17.000000000 +0900
+++ pandoc-1.9.1.2/templates/default.latex	2012-03-23 18:18:31.619693151 +0900
@@ -1,19 +1,20 @@
 \documentclass[$if(fontsize)$$fontsize$,$endif$$if(lang)$$lang$,$endif$]{$documentclass$}
 \usepackage{amssymb,amsmath}
-\usepackage{ifxetex,ifluatex}
+%\usepackage{ifxetex,ifluatex}
+\usepackage{ifxetex}
 \ifxetex
   \usepackage{fontspec,xltxtra,xunicode}
   \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
   \newcommand{\euro}{€}
 \else
-  \ifluatex
-    \usepackage{fontspec}
-    \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
-    \newcommand{\euro}{€}
-  \else
+%  \ifluatex
+%    \usepackage{fontspec}
+%    \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
+%    \newcommand{\euro}{€}
+%  \else
     \usepackage[utf8]{inputenc}
     \usepackage{eurosym}
-  \fi
+%  \fi
 \fi
 $if(natbib)$
 \usepackage{natbib}
