--- pandoc-1.8.2.1/pandoc.cabal~	2011-08-02 03:02:52.000000000 +0900
+++ pandoc-1.8.2.1/pandoc.cabal	2012-01-26 11:44:45.097493599 +0900
@@ -168,7 +168,7 @@
   Default:       True
 Flag highlighting
   Description:   Compile in support for syntax highlighting of code blocks.
-  Default:       False
+  Default:       True
 Flag executable
   Description:   Build the pandoc executable.
   Default:       True
