--- pandoc-2.9.2.1/cabal.project~	2020-03-18 06:37:30.000000000 +0800
+++ pandoc-2.9.2.1/cabal.project	2021-04-30 16:10:05.199110898 +0800
@@ -1,4 +1,4 @@
-packages: pandoc.cabal
+packages: . pandoc-citeproc-0.17.0.2
 
 package pandoc
   flags: +embed_data_files -trypandoc
@@ -7,8 +7,3 @@
 package pandoc-citeproc
   flags: +embed_data_files +bibutils -unicode_collation -test_citeproc -debug
   ghc-options: -j +RTS -A64m -RTS
-
-source-repository-package
-    type: git
-    location: https://github.com/jgm/pandoc-citeproc
-    tag: 0.17
