--- pandoc-1.9.1.1/templates/default.latex~	2012-02-12 10:08:47.000000000 +0900
+++ pandoc-1.9.1.1/templates/default.latex	2012-02-13 13:19:32.020231122 +0900
@@ -1,16 +1,17 @@
 \documentclass[$if(fontsize)$$fontsize$,$endif$]{$documentclass$}
 \usepackage{amssymb,amsmath}
-\usepackage{ifxetex,ifluatex}
+%\usepackage{ifxetex,ifluatex}
+\usepackage{ifxetex}
 \ifxetex
   \usepackage{fontspec,xltxtra,xunicode}
   \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
 \else
-  \ifluatex
-    \usepackage{fontspec}
-    \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
-  \else
+%  \ifluatex
+%    \usepackage{fontspec}
+%    \defaultfontfeatures{Mapping=tex-text,Scale=MatchLowercase}
+%  \else
     \usepackage[utf8]{inputenc}
-  \fi
+%  \fi
 \fi
 $if(natbib)$
 \usepackage{natbib}
