--- pandoc-1.8.2.1/src/markdown2pdf.hs~	2011-08-02 03:02:52.000000000 +0900
+++ pandoc-1.8.2.1/src/markdown2pdf.hs	2011-12-22 12:08:02.528735085 +0900
@@ -173,8 +173,12 @@
 
 saveOutput :: FilePath -> FilePath -> IO ()
 saveOutput input output = do
-  copyFile (encodeString input) (encodeString output)
-  UTF8.hPutStrLn stderr $! "Created " ++ output
+  inExists <- doesFileExist (encodeString input)
+  when inExists $
+    copyFile (encodeString input) (encodeString output)
+  outExists <- doesFileExist (encodeString input)
+  when outExists $
+    UTF8.hPutStrLn stderr $! "Created " ++ output
 
 main :: IO ()
 main = bracket
